library/ssl_tls.c

```diff
@@ -6370,6 +6370,12 @@ static uint16_t ssl_preset_default_sig_algs[] = {
     MBEDTLS_TLS13_SIG_ECDSA_SECP521R1_SHA512,
 #endif /* MBEDTLS_SHA512_C && MBEDTLS_ECP_DP_SECP521R1_ENABLED */
 #endif /* MBEDTLS_ECDSA_C */
+
+    /* RSA algorithms */
+#if defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
+    MBEDTLS_TLS13_SIG_RSA_PSS_RSAE_SHA256,
```

Comment:
Name should follow up name schema. to align with #5166 
```suggestion
    MBEDTLS_TLS1_3_SIG_RSA_PSS_RSAE_SHA256,
```

Reply:
Cause all of the macros in ssl_preset_default_sig_algs are TLS1_3, and we leave all of the related tasks into #4998, let's keep it here.  And if need, I can create another PR to change this type.